presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java

```diff
@@ -92,8 +93,9 @@ else if ("Mac OS X".equals(osName)) {
 
     private static void verifyJavaVersion()
     {
-        if (Runtime.version().feature() < 11) {
-            failRequirement("Presto requires Java 11+ (found %s)", Runtime.version());
+        Version required = Version.parse("11.0.7");
+        if (Runtime.version().compareTo(required) < 0) {
```

Comment:
it's preexisting, but shall we treat `Runtime.version()` as the official version?
in some other places we use `StandardSystemProperty.JAVA_VERSION.value()`. Intentional?